src/Juvix/Compiler/Pipeline.hs

```diff
@@ -119,21 +119,33 @@ upToGeb ::
 upToGeb spec =
   upToStoredCore >>= \Core.CoreResult {..} -> storedCoreToGeb spec _coreResultModule
 
+upToAnoma ::
+  (Members '[HighlightBuilder, Reader Parser.ParserResult, Reader EntryPoint, Reader Store.ModuleTable, Files, NameIdGen, Error JuvixError, GitClone, PathResolver] r) =>
+  Sem r (Nockma.Term Natural)
+upToAnoma = upToStoredCore >>= \Core.CoreResult {..} -> Nockma.TermCell <$> coreToAnoma _coreResultModule
```

Comment:
It should be `storedCoreToAnoma` (using `storedCoreToTree`) instead of `coreToAnoma`. Otherwise, we're doing the `Core.toStored` transformations twice.

Reply:
Thank you. Done